R/estimate_cox.R

```diff
@@ -0,0 +1,181 @@
+#' @title Wrapper for Cox Proportional Hazard analysis
+#'
+#' @description This function is a wrapper around \code{survival::coxph} to fit a Cox Proportional Hazards Regression model, assuming right-censored data.
+#'    The function expects that the data has been filtered on the parameter (PARAM/PARAMCD) of interest.
+#'    The result is an object of class \code{summary.coxph} which can be used in downstream functions and methods that rely on the \code{summary.coxph} class.
+#'
```

Comment:
#'    By default:
#'    \itemize{
#'      \item{A two-sided pointwise 0.95 confidence interval is estimated using a log transformation (conf.type = "log").}
#'      \item{The Efron approximation is used for tie handling.}
#'    }

@AlexandraP-21 Should we enforce some other SAS defaults? What are some of the other differences with SAS? Should we document them in this description as well? I believe it might be important for Pharma to understand those without investigating it themselves as we did. We have a presentation about those differences ;)